tests/station11.test.tsx

```diff
@@ -46,10 +46,16 @@ describe('<App />', () => {
 
   it('shows the list of image when the button is clicked', async () => {
     const res = await render(<App />)
-    const button = res.container.querySelector('button')
+    const buttons = Array.from(res.container.querySelectorAll('button'))
+    const button = buttons.find(r =>
+      (['表示', 'Show'] as any[]).includes(r.innerHTML?.trim() ?? ''),
+    )
 
     expect(button).toBeTruthy()
 
+    if(!button)return;
```

Comment:
```suggestion
    if (!button) {
      return
    }
```
[MUST]
インデントを整えること

[nits]
書き方を他の箇所に合わせた方が良い。
認知負荷が上がるので。